lighthouse-core/config/config.js

```diff
@@ -437,9 +437,15 @@ class Config {
       assertValidPluginName(configJSON, pluginName);
 
       const pluginPath = resolveModule(pluginName, configDir, 'plugin');
+
       const rawPluginJson = require(pluginPath);
       const pluginJson = ConfigPlugin.parsePlugin(rawPluginJson, pluginName);
 
+      if (rawPluginJson['localePath']) {
+        const pluginParentPath = 
+          pluginPath.replace(new RegExp(`/${pluginName}/.*`,'g'), '');
+        i18n.mergeLocales(rawPluginJson['localePath'], pluginParentPath);
```

Comment:
IMO it'd be better to read the files from disk here in config. and let the new i18n methods just deal with objects